.github/workflows/linting.yml

```
@@ -165,12 +165,11 @@ jobs:
     name: Linting Summary
     runs-on: ubuntu-22.04
     needs: [isort, shell-check, log-check, ftest-tags, flake8-lint, doxygen, pylint, codespell]
-    if: always()
+    if: ${{ !cancelled() }}
```

Just FYI, generally speaking, you don't need to use `${{ }}` in `if:` expressions as it's implied.  But if the `!` is causing (i.e. YAML) parsing problems you can use `( )` around it:

```suggestion
    if: (!cancelled())
```

Maybe you already know all of this and just chose `${{ }}` over `( )`.  I have tended to use the latter.

---

Yeah, the `!` caused parsing problems. I'm new to GHA :) Let me try that suggestion

---

If this summary works well, I'd like to put it in a reusable workflow, so simple is better

---

Yep, `if: (!cancelled())` worked :) Thanks!